src/rinoh/template.py

```diff
@@ -497,8 +497,10 @@ def __init__(self, name, base=None, template=None, description=None,
         tmpl_cls = self.template
         for attr, value in options.items():
             options[attr] = tmpl_cls.validate_attribute(attr, value, True)
-        super().__init__(name, base=base or self.template, **options)
+        base = base or self.template
+        super().__init__(name, base=base, **options)
         self.description = description
+        self.variables.update(base.variables)
```

Comment:
Why are you doing this? Variables are looked up in the base (see `RuleSet.get_variable()`) when not defined in the current template configuration. Note that your change overwrites this template configuration's variables with those of _base_.

Reply:
This was my attempt to make the unit tests pass unmodified. 

From the information that you provided here it looks like the unit tests in `test_sphinx.py` should no longer be asserting that `paper_size` is in the `variables` dict. When I remove that assertion, the test passes and correctly falls back to `RuleSet.get_variable()` to determine `paper_size`.